src/main/play-doc/operation/DeployingBundlesOps.md

```diff
@@ -80,21 +80,21 @@ In the example above the bundle shorthand expression `reactive-maps-backend-regi
+As part of the resolution process, a list of resolvers will be selected to resolve the given bundle input. This list of resolvers is called resolver chain. Both `conduct load` or `conduct deploy` has a default resolver chain which is used in most situations, as well as an offline resolver chain. When working during `--offline` mode, the offline resolver chain will be used. It is possible to override the default resolver chain, and this is discussed as part of implementing [custom resolver](Implementing-your-own-custom-resolver).
 
 Each resolver declares a list of supported URI schemes. For example, `bintray_resolver` only accepts [bundle shorthand expression](#Bundle-shorthand-expression) and hence `bundle` scheme, while `uri_resolver` accepts `file`, `http`, and `https` scheme.
 
-These are the steps taken during bundle resolution process.
+The following steps are taken during bundle resolution process.
 
 First, a resolver chain is selected. The resolver chain can be either default, offline, or custom.
 
-Next, a list of possible URI schemes is then deduced from the input URI. For example, if `my-app` is given as the input, this is a valid [bundle shorthand expression](#Bundle-shorthand-expression), and thus the `bundle` scheme is applicable to the URI. However if `my-app` also exists in the local path, the `file` scheme is also applicable. In instances, the applicable schemes to `my-app` are `bundle` and `file`.
+Next, a list of possible URI schemes is deduced from the input URI. For example, if `my-app` is given as the input, then the the `bundle` scheme applies to the URI because `my-app` is a valid [bundle shorthand expression](#Bundle-shorthand-expression). If `my-app` also exists in the local path, then the `file` scheme is also applicable. So in the above example, the schemes `bundle` and `file` apply to `my-app`.
 
-The list of possible URI scheme from input is then compared to the list of URI schemes supported by each resolver in the selected chain. Only resolvers which are able to accept URI scheme from input is used during bundle resolution process.
+The list of possible URI schemes is then compared to the list of URI schemes supported by each resolver in the selected chain. Only resolvers which can accept the selected URI schemes are used during the bundle resolution process.
 
-The input is then passed into each remaining resolver in the chain until a successful resolution can be found. The result of a successful resolution is the bundle artefact which will then be cached. If none of the remaining resolver in the chain is able to resolve the input, an error is raised.
+The input is then passed into each remaining resolver in the chain until a successful resolution can be found. The result of a successful resolution is the bundle artifact which will then be cached. If none of the remaining resolvers in the chain can resolve the input, an error is raised.
```

Comment:
Artifact with an `i` is american english which is what we use in our documentation.